include/mdw_dslash5_tensor_core.cuh

```diff
@@ -0,0 +1,475 @@
+#pragma once
+
+#include <color_spinor_field.h>
+#include <color_spinor_field_order.h>
+#include <dslash_quda.h>
+#include <index_helper.cuh>
+#include <inline_ptx.h>
+#include <math_helper.cuh>
+#include <shared_memory_cache_helper.cuh>
+
+#if (__CUDACC_VER_MAJOR__ >= 9 && __COMPUTE_CAPABILITY__ >= 700)
+#include <mma.h>
+
+// The `mma.sync` PTX is only available with or after CUDA 10.1
+#if ((__CUDACC_VER_MAJOR__ == 10 && __CUDACC_VER_MINOR__ >= 1) || (__CUDACC_VER_MAJOR__ > 10))                         \
+  && (__COMPUTE_CAPABILITY__ < 750)
```

Comment:
Replace this with `== 700` (I believe) as opposed to `< 750`. Alternatively, if this also works well on A100, add a `__COMPUTE_CAPABILITY == 800` on top of the original check.